homeassistant/components/generic/config_flow.py

```diff
@@ -413,6 +429,7 @@ def __init__(self, config_entry: ConfigEntry) -> None:
         """Initialize Generic IP Camera options flow."""
         self.config_entry = config_entry
         self.preview_cam: dict[str, Any] = {}
+        self.preview_stream: PreviewStream | None = None
```

Comment:
this appears unused

Reply:
It is needed within `ws_start_preview()` which is built to support `GenericOptionsFlowHandler` in the future. Options flow stream preview will be in a follow on PR.

Reply:
Without this in, mypy gives me the error
```
homeassistant/components/generic/config_flow.py:636: error: Item "GenericOptionsFlowHandler" of "GenericIPCamConfigFlow | GenericOptionsFlowHandler" has no attribute "preview_stream"  [union-attr]
```